### PR TITLE
add basic windows job settings

### PR DESCRIPTION
**Pull request by liufengyun**
add basic windows job settings

- https://github.com/sbt/sbt/issues/5224 happens when running Windows container with HyperV + Java 8 (https://github.com/sbt/sbt/issues/5179).
- Parallel testing crashes Windows container (Java 11, 14, 15)
- Running docker with `--isolation process` crashes the host Windows system.

### CODE REVIEW

**Review by anatoliykmetyuk — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -96,8 +96,35 @@ jobs:
...
+      - name: Git Checkout
+        uses: actions/checkout@v2
+
+      - name: Test
+        run: sbt ";dotty-bootstrapped/compile ;dotty-bootstrapped/test"
+        shell: cmd
+
+      # - name: Scala.js Test
+      #   run: sbt ";sjsJUnitTests/test"
+      #   shell: cmd
```
Comment (sjrd) [inline .github/workflows/ci.yaml:124]: Why is that disabled?

Comment (liufengyun) [reply .github/workflows/ci.yaml:124]: It causes a failure: https://github.com/lampepfl/dotty/runs/1225047885?check_suite_focus=true

Do you see clue from the message?

Comment (sjrd) [reply .github/workflows/ci.yaml:124]: Yeah ^^
```
Caused by: java.io.IOException: Cannot run program "node": CreateProcess error=2, The system cannot find the file specified
```
We need to install Node.js ;)

Comment (liufengyun) [reply .github/workflows/ci.yaml:124]: Cool, thanks. I'm making a PR to enable it.

**Pull request merged**